packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1660,6 +1644,10 @@ class BuildOwner {
       }
       return true;
     });
+    if (element._inDirtyList) {
+      markNeedsToResortDirtyElements();
```

Comment:
should we just inline this now? it's the only call-site, and it's a weird thing to have in a public api